hummingbot/connector/connector/balancer/balancer_connector.py

```diff
@@ -48,6 +48,7 @@ class BalancerConnector(ConnectorBase):
     """
     API_CALL_TIMEOUT = 10.0
     POLL_INTERVAL = 60.0
+    UPDATE_ORDER_STATUS_MIN_INTERVAL = 10.0
```

Comment:
I don't think we need this min interval, it's there in other connectors which have web socket updates, so the _update_order_status is used as a backup. So for this connector, the _update_order_status should be called every poll_interval.
Also the poll_interval of 60 seconds looks a bit too long to me.

Reply:
I separated the tx poll order update from the main _status_polling_loop() into _tx_polling_loop(). This let us keep 2 time intervals (1 for balance, another for faster tx polling).

Reply:
I refactored this further to use the single loop.